### PR TITLE
SDK-870 swagger docs

### PR DESCRIPTION
**Pull request by sergeyspryker**
https://spryker.atlassian.net/browse/SDK-870

### CODE REVIEW

File: src/Presentation/RestApi/Controller/v1/SdkInitSdkController.php [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+ */
+
+namespace SprykerSdk\Sdk\Presentation\RestApi\Controller\v1;
+
+use OpenApi\Annotations as OA;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+class SdkInitSdkController
```
Comment (vol4onok) [inline src/Presentation/RestApi/Controller/v1/SdkInitSdkController.php:15]: how to use this for dynamic routes. one controller and few tasks (routes)

File: src/Infrastructure/Event/Request/JsonRequestListener.php [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    {
+        if (!$event->isMainRequest()) {
+            return;
+        }
+
+        $request = $event->getRequest();
+
+        if (
+            is_resource($request->getContent())
+            || $request->getContent() === ''
```
Comment (vol4onok) [inline src/Infrastructure/Event/Request/JsonRequestListener.php:32]: Does it a filter request with body?

**Review by vol4onok — APPROVED** [review]

File: config/routes/nelmio_api_doc.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# Expose your documentation as JSON swagger compliant
```
Comment (pavelmaksimov25) [inline config/routes/nelmio_api_doc.yaml:1]: Please commented lines

File: src/Infrastructure/Event/Request/JsonRequestListener.php [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+    public function onKernelRequest(RequestEvent $event): void
+    {
+        if (!$event->isMainRequest()) {
+            return;
+        }
+
+        $request = $event->getRequest();
+
+        if (
+            is_resource($request->getContent())
```
Comment (pavelmaksimov25) [inline src/Infrastructure/Event/Request/JsonRequestListener.php:31]: Please, extract to the separate method `isApplicabe` or `supports` to increase readability.

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**